doc/book/methods.md

```diff
@@ -1,62 +1,39 @@
-
-Method signature                               | Description
----------------------------------------------- | -----------
-`addChild(string|RoleInterface $child) : void` | Add a child role to the current instance.
-`addPermission(string $name) : void`           | Add a permission for the current role.
-`getName() : string`                           | Retrieve the name assigned to this role.
-`hasPermission(string $name) : bool`           | Does the role have the given permission?
-`setParent(RoleInterface $parent) : void`      | Assign the provided role as the current role's parent.
-`addParent(RoleInterface $parent) : Role`      | Add a parent role to the current instance.
-`getParent() null|RoleInterface|array`         | Retrieve the current role's parent, or array of parents if more that one exists.
+## `Zend\Permissions\Rbac\Role`
+
+The `Role` provides the base functionality required by the `RoleInterface`.
+
+Method signature                          | Description
+------------------------------------------| -----------
+`__construct(string $name) : void`        | Create a new instance with the provided name.
+`getName() : string`                      | Retrieve the name assigned to this role.
+`addPermission(string $name) : void`      | Add a permission for the current role.
+`hasPermission(string $name) : bool`      | Does the role have the given permission?
+`addChild(RoleInterface $child) : Role`   | Add a child role to the current instance.
+`getChildren() : RoleInterface[]`         | Get all the children roles.
+`addParent(RoleInterface $parent) : Role` | Add a parent role to the current instance.
+`getParents() : RoleInterface[]`          | Get all the parent roles.
 
 ## `Zend\Permissions\Rbac\AssertionInterface`
 
 Custom assertions can be provided to `Rbac::isGranted()` (see below); such
 assertions are provided the `Rbac` instance on invocation.
 
-Method signature            | Description
---------------------------- | -----------
-`assert(Rbac $rbac) : bool` | Given an RBAC, determine if permission is granted.
+Method signature                                                     | Description
+-------------------------------------------------------------------- | -----------
+`assert(Rbac $rbac, RoleInterface $role, string $permission) : bool` | Given an RBAC, a role, and a permission, determine if permission is granted.
 
 ## `Zend\Permissions\Rbac\Rbac`
 
 `Rbac` is the object with which you will interact within your application in
-order to query for permissions. It extends `AbstractIterator`.
+order to query for permissions.
 
 Method signature                                                            | Description
 --------------------------------------------------------------------------- | -----------
 `addRole(string|RoleInterface $child, array|RoleInterface $parents = null)` | Add a role to the RBAC. If `$parents` is non-null, the `$child` is also added to any parents provided.
```

Comment:
Backslash is required for pipe:

`addRole(string\|RoleInterface $child, array\|RoleInterface $parents = null)`

Reply:
Problem is the rendering inside tables.